tests/sentry/workflow_engine/handlers/action/notification/test_issue_alert.py

```diff
@@ -305,3 +332,28 @@ def test_build_rule_action_blob_no_priority(self):
             "team": "team789",
             "priority": "",
         }
+
+
+class TestGithubIssueAlertHandler(BaseWorkflowTest):
+    def setUp(self):
+        super().setUp()
+        self.handler = TicketingIssueAlertHandler()
+
+        self.action_data = pop_keys_from_data_blob(GITHUB_ACTION_DATA_BLOBS, Action.Type.GITHUB)
+
+    def test_build_rule_action_blob(self):
+        """Test that build_rule_action_blob creates correct Github action data"""
+        for action_blob, expected in zip(self.action_data, GITHUB_ACTION_DATA_BLOBS):
```

Comment:
It took me a minute to figure out exactly what was happening in this test 😅 Instead of iterating over each blob and filtering its contents in the `pop_keys_from_data_blob` function, then zipping it together with the original blob, I think it might be beneficial to operate on each blob individually.

It's also a bit more composable this way, allowing us to chain other assertions/transformations down the line as well if they come up.